cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -416,10 +416,6 @@ spec:
             - name: REMOTE_WRITE_ENABLED
               value: "true"
             {{- end }}
-            - name: GOGC
-              value: "60"
-            - name: GODEBUG
-              value: "madvdontneed=1"
```

Comment:
Are we removing this because its now default behavior in Go? Just want to make sure.

Reply:
GODEBUG, yes. GOGC is 100 by default.